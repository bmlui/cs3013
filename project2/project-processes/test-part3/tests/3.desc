Test && operator, first condition true
