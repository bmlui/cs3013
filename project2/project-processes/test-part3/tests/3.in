/usr/bin/true && echo it works
exit

