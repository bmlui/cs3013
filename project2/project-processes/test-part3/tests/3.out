test-part3$ /usr/bin/true && echo it works 
it works 
test-part3$ exit
